app/dashboard/views.py

```diff
@@ -475,10 +475,14 @@ def send_tip_2(request):
 
 
 @staff_member_required
-def onboard(request):
+def contributor_onboard(request):
     """Handle displaying the first time user experience flow."""
-    params = {'title': _('Onboarding Flow')}
-    return TemplateResponse(request, 'onboard.html', params)
+    params = {
+        'title': _('Onboarding Flow'),
+        'steps': ['github', 'metamask', 'avatar', 'skills'],
```

Comment:
Pretty much what I was thinking.  Ideally, we'd take this a step ( 😁) further to allow us to provide the series of steps defined somewhere on the BE based on the user type or other criteria...  Thoughts?

Reply:
> Pretty much what I was thinking.

Sauron is pleased :P

Couldn't we just add that logic in here ? 
`profile.getUserType()` --> based on that we specify flow ! 
Any benefits your approach would offer over the one you laid out there ? (I probably didn't get this 😅 )